src/pages/cody.tsx

```diff
@@ -103,60 +184,355 @@ const CodyPage: FunctionComponent = () => {
+                                                />
+                                                <div className="absolute bottom-0 h-[1px] w-[314px] bg-black opacity-[13%]" />
+                                            </div>
+                                            <div className="relative">
+                                                <div className="absolute h-[1px] w-[244px] bg-black opacity-[13%]" />
+                                                <CodyAbility
+                                                    src="/assets/cody/file-question-icon.svg"
+                                                    title="Explain Code"
+                                                    subTitle="Understand the open project or file better"
+                                                />
+                                                <div className="absolute bottom-0 right-[30px] h-[1px] w-[428px] bg-black opacity-[13%]" />
+                                            </div>
+                                            <div className="relative">
+                                                <div className="absolute h-[1px] w-[368px] bg-black opacity-[13%]" />
+                                                <CodyAbility
+                                                    src="/assets/cody/open-book-icon.svg"
+                                                    title="Document Code"
+                                                    subTitle="Add comments to file or selection"
+                                                />
+                                                <div className="absolute bottom-0 h-[1px] w-[277px] bg-black opacity-[13%]" />
+                                            </div>
+                                            <div className="relative">
+                                                <div className="absolute h-[1px] w-[368px] bg-black opacity-[13%]" />
+                                                <CodyAbility
+                                                    src="/assets/cody/hammer-icon.svg"
+                                                    title="Generate Unit Tests"
+                                                    subTitle="Create tests for the open file"
+                                                />
+                                                <div className="absolute bottom-0 h-[1px] w-[368px] bg-black opacity-[13%]" />
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                        <div className='xl:bg-start w-[812px] overflow-hidden bg-none xl:border-b xl:border-r xl:border-[#DBE2F0] xl:bg-[url("/assets/cody/syntax-highlighter-bg.svg")] xl:bg-cover'>
```

Comment:
There is a gap above this "syntax-highlighter-bg.svg" that shouldn't be there. It should be flush with the element above it. 
<img width="329" alt="Screenshot 2024-09-25 at 8 36 16 AM" src="https://github.com/user-attachments/assets/ff9fdaea-7bad-43c5-a9db-4c7fa83c289c">
Can you adjust?